cookbooks/steinberger/doc/steinberger.tex

```diff
@@ -0,0 +1,157 @@
+\subsubsection{Convection using a pressure--temperature look-up table and the rheology of Steinberger and Calderwood (2006)}
+\label{sec:cookbooks-steinberger}
+
+\textit{This section was contributed by Juliane Dannberg and Ren{\'e} Gassm{\"o}ller.}
+
+In this cookbook we will go one step further from the last one and set up a fully compressible mantle convection model using the projected density approximation (where the density is interpolated onto the finite element grid to compute the density gradients in the mass conservtaion equation rather than approximating these gradients using a reference profile or tempertaure/pressure derivatives of the density, see \cite{gassmoller2020formulations}). To compute the material properties, we read in a look-up table of material properties in dependence of temperature and pressure computed using a mineral physics software (in this case, Perple\_X, \cite{connolly2005computation}). The table is based on the thermodynamic database by \cite{stixrude2011thermodynamics} and a pyrolitic composition \cite{ringwood1988nature}. Compared to a 1D profile, a temperature--pressure look-up table has the advantage that material properties are accurate not only around one reference adiabat, but also for strongly deviating pressures and temperatures. This is particularly important at phase transitions, because their depth depends on the temperature and pressure. 
+
+This cookbook also demonstrates how to read in a viscosity profile from a data file. Specifically, we use the profile and lateral viscosity variations due to temperature from \cite{stca06}, which are based on mineral physics constraints and surface observations. 
+
+In addition, this cookbook shows the use of periodic boundary conditions. 
+
+\paragraph{Geometry and periodic boundaries.}
+The model setup is a quarter spherical shell with periodic side boundaries. The inner and outer radius are 3481~km and 6371~km, respectively, so that the mantle is 2900~km deep. In the same section of the input file, we also need to specify that the model should have periodic boundaries in angular ($\phi$) direction:  
```

Comment:
```suggestion
The model setup is a quarter spherical shell with periodic side boundaries. The inner and outer radius are \SI{3481}{km} and \SI{6371}{km}, respectively, so that the mantle is \SI{2900}{km} deep. In the same section of the input file, we also need to specify that the model should have periodic boundaries in angular ($\phi$) direction:  
```